Dockerfile

```diff
@@ -103,6 +109,11 @@ RUN pip install --no-cache-dir -e .[test] && rm -rf ./build
 
 # Container with only cuda runtime, half the size of dev container
 FROM nvidia/cuda:11.6.0-runtime-ubuntu20.04 as timemachine
+ARG LIBXRENDER_VERSION
+ARG LIBXEXT_VERSION
+RUN apt-get update && apt-get install --no-install-recommends -y libxrender1=${LIBXRENDER_VERSION} libxext-dev=${LIBXEXT_VERSION} \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Not sure if this is possible, but (assuming `tm_base_env` has more stuff than we need for the stripped-down image), could this be moved to a common base target extended by both?

Reply:
I don't think it is possible because they have different base images. I had considered copying the libraries from the original container, but seemed safer to just install in the new container.